addons/website_sale/views/templates.xml

```diff
@@ -1351,149 +1344,142 @@
-                                    <t t-set="icon" t-value="attachment_sudo.type == 'url' and 'fa-link' or 'fa-download'"/>
-                                    <div>
-                                        <a t-att-href="'/shop/' + slug(product) + '/document/' + str(document_sudo.id)" t-att-target="target">
-                                            <i t-att-class="'fa ' + icon"/>
-                                            <t t-out="attachment_sudo.name"/>
-                                        </a>
+                                            </small>
+                                        </div>
+                                        <t name="variant_info">
+                                            <input
+                                                type="hidden"
+                                                class="o_not_editable product_id"
+                                                name="product_id"
+                                                t-att-value="product_variant.id"
+                                            />
+                                            <input
+                                                type="hidden"
+                                                class="o_not_editable product_template_id"
+                                                name="product_template_id"
+                                                t-att-value="product.id"
+                                            />
+                                            <input
+                                                t-if="product.public_categ_ids.ids"
+                                                type="hidden"
+                                                class="product_category_id"
+                                                name="product_category_id"
+                                                t-att-value="product.public_categ_ids.ids[0]"
+                                            />
+                                            <input
+                                                type="hidden"
+                                                name="product_type"
+                                                t-att-value="product.type"
+                                            />
+                                            <t t-call="website_sale.variants">
+                                                <t t-set="ul_class" t-valuef="flex-column" />
+                                                <t t-set="parent_combination" t-value="None" />
+                                                <t t-set="combination" t-value="combination_info['combination']"/>
+                                            </t>
+                                        </t>
+                                        <t t-if="website.product_page_image_width in ['50_pc', '66_pc']" t-call="website_sale.cta_wrapper"/>
```

Comment:
If you define as suggested above a `is_fullwidth_content` variable you will be able to use 
`not is_fullwidth_content` for the 50_pc & 66_pc cases